pkg/config/config.go

```diff
@@ -155,3 +155,21 @@ func LoadConfFromCache(args *skel.CmdArgs) (*sriovtypes.NetConf, string, error)
 
 	return netConf, cRefPath, nil
 }
+
+// GetMacAddressForResult return the mac address we should report to the CNI call return object
+// if the device is on kernel mode we report that one back
+// if not we check the administrative mac address on the PF
+// if it is set and is not zero, report it.
+func GetMacAddressToReport(netConf *sriovtypes.NetConf) string {
```

Comment:
i like the name in the comment better :) (GetMacAddressForResult)

that said, please align one to the other if u prefer GetMacAddressToReport

Reply:
done